src/server/server.ts

```diff
@@ -120,7 +120,7 @@ app.all('*', (request, response, next) => {
   next();
 });
 
-app.use('/public', express.static(path.resolve(__dirname, '../public')));
+app.use('/assets', express.static(path.resolve(__dirname, '../assets')));
```

Comment:
Even though `/assets` is already defined the line after, the way express works is:
If it can find the file: great, otherwise search in the next folder.

So fonts are first served, and compiled javascript will go through two hops. This has no effect in production as it's served as static files from the CDN.